### PR TITLE
test(interaction): add data cell multi selection test

### PR DESCRIPTION
**Pull request by lijinke666**
### 👀 PR includes

🔧 Chore

- [x] Test case
- [ ] Docs / demos update
- [ ] CI / workflow
- [ ] Release version
- [ ] Other ()

### 📝 Description

增加数值单元格多选测试

### 🖼️ Screenshot

| Before                         | After                         |
| ------------------------------ | ------------------------------ |
| ❌                             | ✅                              |

### 🔍 Self Check before Merge

- [x] Add or update relevant Docs.
- [x] Add or update relevant Demos.
- [x] Add or update relevant TypeScript definitions.

### CODE REVIEW

**Comment by lijinke666** [comment]
写测试 mock 比写代码难多了...

**Comment by xingwanying** [comment]
> 写测试 mock 比写代码难多了...

那可不……代码只管摞起来

File: packages/s2-core/__tests__/unit/interaction/data-cell-multi-selection-spec.ts [hunk 1]
```diff
@@ -0,0 +1,192 @@
+import { createFakeSpreadSheet, sleep } from 'tests/util/helpers';
+import { Event as GEvent } from '@antv/g-canvas';
```
Comment (xingwanying) [inline packages/s2-core/__tests__/unit/interaction/data-cell-multi-selection-spec.ts:2]: 这俩lint解决下？

**Review by xingwanying — DISMISSED** [review]
LG

**Review by xingwanying — APPROVED** [review]
LG

**Pull request merged**